spec/helpers/argo_helper_spec.rb

```diff
@@ -24,25 +24,27 @@
       @apo_id = 'druid:hv992ry2431'
       @object = instantiate_fixture('druid_zt570tx3016', Dor::Item)
       @doc = SolrDocument.new({'id' => @item_id, SolrDocument::FIELD_APO_ID => [@apo_id]})
-      @usr = double()
-      allow(@usr).to receive(:is_admin).and_return(true)
-      allow(@usr).to receive(:groups).and_return([])
-      allow(@usr).to receive(:is_manager).and_return(false)
-      allow(@usr).to receive(:roles).with(@apo_id).and_return([])
-      allow(Dor::WorkflowService).to receive(:get_active_lifecycle).and_return(true)
-      allow(Dor::WorkflowService).to receive(:get_lifecycle).and_return(true)
-      allow(helper).to receive(:current_user).and_return(@usr)
       allow(@object).to receive(:can_manage_item?).and_return(true)
       allow(@object).to receive(:pid).and_return(@item_id)
       desc_md = double(Dor::DescMetadataDS)
       id_md   = double(Dor::DescMetadataDS)
-      apo     = double()
+      apo = double(Dor::AdminPolicyObject)
+      allow(apo).to receive(:pid).and_return(@apo_id)
       allow(desc_md).to receive(:new?).and_return(true)
       allow(id_md).to receive(:ng_xml).and_return(Nokogiri::XML('<identityMetadata><identityMetadata>'))
       allow(apo).to receive(:pid).and_return(@apo_id)
       allow(@object).to receive(:datastreams).and_return({'contentMetadata' => nil, 'descMetadata' => desc_md, 'identityMetadata' => id_md})
       allow(@object).to receive(:admin_policy_object).and_return(apo)
       allow(Dor).to receive(:find).with(@item_id).and_return(@object)
+      allow(Dor::WorkflowService).to receive(:get_active_lifecycle).and_return(true)
+      allow(Dor::WorkflowService).to receive(:get_lifecycle).and_return(true)
+      # User mocks
+      @usr = double()
+      allow(@usr).to receive(:is_admin).and_return(true)
+      allow(@usr).to receive(:groups).and_return([])
+      allow(@usr).to receive(:is_manager).and_return(false)
+      allow(@usr).to receive(:roles).with(@apo_id).and_return([])
+      allow(helper).to receive(:current_user).and_return(@usr)
```

Comment:
Simply rearranged the code to clarify what is going on here.  Although some of this might be replaced with an `admin_user` from spec_helper, I did not want to mess with this spec too much.  I get the impression a lot of this might have been done before `instantiate_fixture` arrived on the scene, but I didn't want to change it.